src/server.cc

```diff
@@ -167,6 +167,8 @@ Status Server::Start() {
     }
   });
 
+  std::cout << "Ready to accept connections\n";
```

Comment:
```suggestion
  LOG(INFO) << "Ready to accept connections";
```

Reply:
However TCL tests handle log files hacky and I tend to send this information to stdout now.